pkg/operation/botanist/logging.go

```
@@ -20,156 +20,34 @@ import (
-	if b.Shoot.GetPurpose() == gardencorev1beta1.ShootPurposeTesting || !gardenletfeatures.FeatureGate.Enabled(features.Logging) {
-		return common.DeleteLoggingStack(ctx, b.K8sSeedClient.Client(), b.Shoot.SeedNamespace)
-	}
-
-	var (
-		kibanaCredentials                      = b.Secrets["kibana-logging-sg-credentials"]
-		kibanaUserIngressCredentialsSecretName = "logging-ingress-credentials-users"
-		sgKibanaUsername                       = kibanaCredentials.Data[secrets.DataKeyUserName]
-		sgKibanaPassword                       = kibanaCredentials.Data[secrets.DataKeyPassword]
-		sgKibanaPasswordHash                   = kibanaCredentials.Data[secrets.DataKeyPasswordBcryptHash]
-		basicAuth                              = utils.EncodeBase64([]byte(fmt.Sprintf("%s:%s", sgKibanaUsername, sgKibanaPassword)))
-
-		sgCuratorPassword     = b.Secrets["curator-sg-credentials"].Data[secrets.DataKeyPassword]
-		sgCuratorPasswordHash = b.Secrets["curator-sg-credentials"].Data[secrets.DataKeyPasswordBcryptHash]
-
-		sgUserPasswordHash  = b.Secrets[kibanaUserIngressCredentialsSecretName].Data[secrets.DataKeyPasswordBcryptHash]
-		sgAdminPasswordHash = b.Secrets[common.KibanaAdminIngressCredentialsSecretName].Data[secrets.DataKeyPasswordBcryptHash]
-
-		userIngressBasicAuth  = utils.CreateSHA1Secret(b.Secrets[kibanaUserIngressCredentialsSecretName].Data[secrets.DataKeyUserName], b.Secrets[kibanaUserIngressCredentialsSecretName].Data[secrets.DataKeyPassword])
-		adminIngressBasicAuth = utils.CreateSHA1Secret(b.Secrets[common.KibanaAdminIngressCredentialsSecretName].Data[secrets.DataKeyUserName], b.Secrets[common.KibanaAdminIngressCredentialsSecretName].Data[secrets.DataKeyPassword])
-		ingressBasicAuth      string
-
-		sgFluentdPasswordHash string
-	)
-
-	userIngressBasicAuthDecoded, err := utils.DecodeBase64(userIngressBasicAuth)
+	err := common.DeleteOldLoggingStack(ctx, b.K8sSeedClient.Client(), b.Shoot.SeedNamespace)
 	if err != nil {
-		return err
+		return nil
```

Why not return the error? If there is a valid reason, would you mind adding a short comment to explain? Otherwise somebody might feel compelled to fix this in the future.

---

just a bug, fixed now :)